scripts/cronish.sh

```diff
@@ -3,7 +3,7 @@
 while true
 do
   echo "Running 'drush cron'..."
-  drush --root=$HOME/web core:cron
+  drush --root=$HOME/web core:cron --uri
```

Comment:
Please update to 
`drush --root=$HOME/web core:cron --uri=https://vote-drupal-test.app.cloud.gov`